girder/stemserver_plugin/models/stemimage.py

```diff
@@ -48,7 +48,19 @@ def create(self, user, file_id=None, file_path=None, public=False):
         if file_id:
             stem_image['fileId'] = file_id
         elif file_path:
-            stem_image['filePath'] = file_path
+            # Only admin users can do this
+            admin = user.get('admin', False)
+            if admin is not True:
+                raise RestException('Only admin users can use a filePath',
+                                    code=403)
+
+            item = self._get_import_stem_item(user)
+            name = self._get_import_unique_file_name(item)
+            assetstore = self._get_assetstore()
+
+            adapter = FilesystemAssetstoreAdapter(assetstore)
+            f = adapter.importFile(item, file_path, user, name)
+            stem_image['fileId'] = f['_id']
```

Comment:
It is good to lock this down now. However, in the future, we make create a 'stem-admin' group rather than tying this to the users that can administer the Girder instance.

Reply:
Okay.